data.sql

```diff
@@ -10,4 +10,65 @@ INSERT INTO animals (id, name, date_of_birth, escape_attempts, neutered, weight_
 VALUES (3, 'Pikachu', '2021-01-07', 1, false, 15.04);
 
 INSERT INTO animals (id, name, date_of_birth, escape_attempts, neutered, weight_kg)  
-VALUES (4, 'Devimon', '2017-05-12', 5, true, 11);
\ No newline at end of file
+VALUES (4, 'Devimon', '2017-05-12', 5, true, 11);
+
+INSERT INTO animals (id, name, date_of_birth, escape_attempts, neutered, weight_kg)  
+VALUES (5, 'Charmander', '2020-02-08', 0, false, -11);
+
+INSERT INTO animals (id, name, date_of_birth, escape_attempts, neutered, weight_kg)  
+VALUES (6, 'Plantmon', '2022-11-15', 2, true, -5.7);
+
+INSERT INTO animals (id, name, date_of_birth, escape_attempts, neutered, weight_kg)  
+VALUES (7, 'Squirtle', '1993-04-02', 3, false, -12.13);
+
+INSERT INTO animals (id, name, date_of_birth, escape_attempts, neutered, weight_kg)  
+VALUES (8, 'Angemon', '2005-06-12', 1, true, -45);
+
+INSERT INTO animals (id, name, date_of_birth, escape_attempts, neutered, weight_kg)  
+VALUES (9, 'Boarmon', '2005-06-07', 7, true, 20.4);
+
+INSERT INTO animals (id, name, date_of_birth, escape_attempts, neutered, weight_kg)  
+VALUES (10, 'Blossom', '1998-10-13', 3, true, 17);
+
+BEGIN TRANSACTION; -- start transaction
+
+-- sets all items/rows in the species column to unspecified
+UPDATE animals SET species = 'unspecified';
+
+--verify the changes were made
+SELECT * FROM animals;
+
+-- Undo changes, species column is now empty
+ROLLBACK;
+
+BEGIN;
+
+UPDATE animals SET species = 'digimon' WHERE name LIKE '%mon';
+
+UPDATE animals SET species = 'pokemon' WHERE species IS NULL;
+
+COMMIT;
+
+SELECT name, species FROM animals;
```

Comment:
Please move all these queries written to update the tables to `queries.sql` as everything that is a query that helps to analyze data should be saved in the `queries.sql` file.